src/inmanta/const.py

```diff
@@ -77,6 +77,18 @@ class DeprecatedResourceState(str, Enum):
     processing_events = "processing_events"
 
 
+"""
+The following consts are considered to be part of the stable API.
+Modifying them may consist a breaking change for some libraries:
+    - UNDEPLOYABLE_STATES
+    - UNDEPLOYABLE_NAMES
+    - TRANSIENT_STATES
+    - NOT_DONE_STATES
+    - DONE_STATES
+    - INITIAL_STATES
+    - TERMINAL_STATES
+    - VALID_STATES_ON_UPDATE
```

Comment:
Is there any of these consts that is more volatile and shouldn't be added to the stable api?